manifests/opsmanager.pp

```diff
@@ -0,0 +1,59 @@
+# This installs Ops Manager. See README.md for more info.
+
+class mongodb::opsmanager (
+  String[1] $user                                = $mongodb::params::opsmanager_user,
+  String[1] $group                               = $mongodb::params::opsmanager_group,
+  Variant[Boolean, String[1]] $ensure            = $mongodb::params::opsmanager_ensure,
+  String[1] $package_name                        = $mongodb::params::opsmanager_package_name,
+  Boolean $package_ensure                        = $mongodb::params::opsmanager_package_ensure,
```

Comment:
Typically we use a `String` for package ensures. This allows `installed` (default), `latest`, `absent` or a specific version. Booleans aren't needed then